shared/chat/inbox/container/remote.js

```diff
@@ -17,14 +57,44 @@ export type RemoteConvMeta = $Diff<
   {onSelectConversation: () => void}
 >
 
-// To cache the list
-const valuesCached = memoize((...vals) => vals.map(v => v))
-
-const metaMapToFirstValues = memoize(metaMap =>
-  metaMap
-    .partialSort(maxShownConversations, (a, b) => b.timestamp - a.timestamp)
-    .filter((_, id) => Constants.isValidConversationIDKey(id))
```

Comment:
Cool

Reply:
it might be faster to just do a partialSort of some multiple, then do the filter. we'l be doing a lot of extra work to filter on a thing we're just going to throw away. I'd assume we'd be safe with maxShownConversations*3 or something. for instance i have 4000 convos, so i'd rather just go through 15 or 100 or something

Reply:
lets do this ('im fine with *100)